ipatests/test_integration/test_sssd.py

```diff
@@ -108,3 +113,45 @@ def test_auth_cache_enabled_when_configured(self, user):
             assert self.is_auth_cached(self.users[user])
             time.sleep(10)
             assert not self.is_auth_cached(self.users[user])
+
+    @contextmanager
+    def filter_user_setup(self, user):
+        sssd_conf_backup = tasks.FileBackup(self.master, paths.SSSD_CONF)
+        filter_user = {'filter_users': '{0}'.format(self.users[user]['name'])}
+        try:
+            if user == 'fakeuser':
+                provider = 'nss'
+            else:
+                provider = 'ipa'
+            tasks.modify_sssd_conf(self.master, self.master.domain.name,
+                                   filter_user, provider=provider)
+            tasks.clear_sssd_cache(self.master)
+            yield
+        finally:
+            sssd_conf_backup.restore()
+
+    @pytest.mark.xfail(
+        osinfo.id == 'fedora' and osinfo.version_number <= (30,),
```

Comment:
The related fix is in sssd-2.2.1
Fedora 29 and up have sssd >= 2.2.2

The test should execute successfully on all fedoras >= 29.